src/consumer/frame.js

```diff
@@ -42,20 +41,20 @@ class Frame extends EventEmitter {
       {
         launch() {
           self.wrapper.setAttribute('data-status', 'launched');
-          self.emit('xfc.launched');
+          self.emit(events.launched);
           return Promise.resolve();
         },
 
         authorized(detail = {}) {
           self.wrapper.setAttribute('data-status', 'authorized');
-          self.emit('xfc.authorized', detail);
+          self.emit(events.authorized, detail);
```

Comment:
Can you explain why you've changed this? How have you tested this?

Reply:
This is just essentially an enum